docs/helm-chart-checks.md

```diff
@@ -75,23 +75,38 @@ This section provides help on the basic usage of Helm chart checks with the podm
   The output is similar to the following example:
   ```
   Verifies a Helm chart by checking some of its characteristics
+  Verifies a Helm chart by checking some of its characteristics
```

Comment:
This can be removed.

Reply:
Removed 8109de456328d19c9a35f3d49687703ae16db8e1